.storybook/themes/news.scss

```diff
@@ -1788,6 +1788,49 @@
 				quad-chain-child-item-empty: (
 					display: contents,
 				),
+				result-list: (
+					padding-bottom: var(--global-spacing-4),
```

Comment:
We should avoid using padding and try to use Stack, or flex or grid layouts to achieve spacing